src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -388,6 +393,7 @@ public PreferenceSettings()
             PythonTemplateFilePath = "";
             IsIronPythonDialogDisabled = false;
             ShowTabsAndSpacesInScriptEditor = false;
+            DefaultPythonEngine = string.Empty;
```

Comment:
Do we want to make PythonEngineVersion enum static so we can refer anywhere with known value options?

Reply:
Do you mean make the `DefaultPythonEngine` preference setting static? I guess that could work, but we also have the system setting, so we'd have to make them both static I guess. It could be an alternative to putting stuff into `ExecutingSession`, which may be cleaner if we can use `InternalsVisibleTo`. Will check that out.